dockerfiles/init/modules/keycloak/files/che/login/register.ftl

```diff
@@ -0,0 +1,76 @@
+<#import "template.ftl" as layout>
```

Comment:
maybe describe the changes with https://github.com/keycloak/keycloak/blob/master/themes/src/main/resources/theme/base/login/register.ftl to better update with new upstream releases.